azure-pipelines.yml

```diff
@@ -165,7 +165,7 @@ stages:
               ignoreDirectories: .packages
 
         - ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
-          - template: /eng/test.yaml
+          - template: /eng/templates/steps/test.yaml
```

Comment:
In one place we have `./eng/...` here `/eng/...` (without a dot). what about to use the same pattern everywhere?

Reply:
I am just looking at that, for some reason the pipeline doesn't accept `/`

- I think `eng/` means "eng in root"
- `./eng` is relative but sometimes we pass a template path as a parameter to another template and that one needs a different relative path

I will have a look whether we can use absolute everywhere

Reply:
I put absolute paths everywhere. Ideally, the paths between the templates would be relative so that you can take the whole template folder and put it somewhere else. However, this clashes a bit with Arcade templates that accept jobs/steps and evaluates that later so it needs to be absolute there.